src/api/courses.test.js

```diff
@@ -13,7 +13,7 @@ afterEach(() => testutil.destroyTestDb())
 
 describe('Courses API', () => {
   test('GET /api/courses', async () => {
-    const request = await requestAsUser(app, 'admin')
+    const request = await requestAsUser(app, 'student')
```

Comment:
Any particular reason this changed? Should we have tests for admins/staff as well?

Reply:
Oh, while I was making some modifications to the course filtering, all of the courses disappeared for students for some reason, which is why I had changed it to 'student' to try and fix my bug. 

But, I do think with the new filtering based on unlisted & whether the student is courseStaff/not, it might be worthwhile for me to write a test case or two for the students as well.